### DISCUSSION TITLE
Wanted to know what this is all about.

### DISCUSSION THREAD

**Discussion by ott0matik** [mercedes-benz/DnA#3640] [General]
I wanted to understand on deeper level what DnA helps with and what are the use cases for it.

**Comment by josipskafar** [comment]
Hi @ott0matik ,
we are using it to:
- achieve transparency on data and its usage
- enable easy consumption of data for different use cases
- enable instant provisioning of tooling required to work with data
- provisioning of links to training topics around tools and data

More info you will find in our wiki: https://github.com/mercedes-benz/DnA/wiki
Also if you need some more deep dive questions - ping me and we can shortly discuss in a Teams call

Best regards,
Josip

**Reply by ott0matik** [reply]
Thank you sir! 
will go through it and ping you if needed!

**Comment by josipskafar** [comment]
closing this - you can open for any new topic
